### PR TITLE
updating the web dev class

### PR DESCRIPTION
**Pull request by Malekalarmash**

### CODE REVIEW

File: public/pages/intro-class.html [hunk 1]
```diff
@@ -237,14 +211,14 @@ <h4>Software Engineering Career Beyond Refcode</h4>
...
+        <p><a href="https://internal.refcode.org/login"class="button mobile-only">Apply Now</a></p>
         <p><a href="https://internal.refcode.org/volunteer" class="button">Volunteer</a></p>
       </div>
       <aside class="floater">
         <p><strong>10 week class</strong> - $200 (scholarships available for refugees and immigrants)</p>
-        <p><strong>June 28-September 6 </strong>, Wednesday nights, 6-9pm</p>
+        <p><strong>August 2-October 4 </strong>, Wednesday nights, 6-9pm</p>
         <p>Students, sign up for the Summer 2023 cohort now!</p>
-        <p><a href="https://internal.refcode.org/waitlist" class="button">Apply Now</a></p>
+        <p><a href="https://internal.refcode.org/login" class="button">Apply Now</a></p>
```
Comment (brentonstrine) [inline public/pages/intro-class.html:221]: We should provide more instructions on what they need to do to apply. If they log in, they'll be confused.

**Review by brentonstrine — APPROVED** [review]
OK to merge, but we shoudl address one comment I left.

File: public/pages/intro-class.html (hunk 1 above)
Comment (Malekalarmash) [reply public/pages/intro-class.html:221]: <h2>Applying for the Class:</h2>
        <p>To apply for the upcoming cohort, follow these steps:</p>
        <ol>
          <li>Click on Apply Now</li>
          <li>Create an account</li>
          <li>Log in using the credentials you used in the previous step</li>
          <li>Go to Web Development 1 assessment</li> 
          <li>Watch the welcome video</li>
          <li>Compelete the tasks </li>
          <li>Come to Refcode to get your assessment reviewed and approved</li>
        </ol>

Comment (Malekalarmash) [reply public/pages/intro-class.html:221]: Did you see this ? I included the steps to enroll

**Pull request merged**